src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/DeleteCaseWorkerProfilesIntegrationTest.java

```diff
@@ -97,7 +98,8 @@ public void deleteCaseWorkerProfileByEmailPattern() {
 
         assertThat(createResponse).containsEntry("http_status", "201 CREATED");
 
-        CaseWorkerProfile createdProfile = caseWorkerProfileRepository.findByEmailId("test.inttest@hmcts.gov.uk");
```

Comment:
add test case to get the CaseWorkerProfile using mixed case email  Test.Inttest@hmcts.gov.uk